addons/l10n_es_edi_facturae/data/account.tax.template.csv

```diff
@@ -0,0 +1,89 @@
+"id","l10n_es_edi_facturae_tax_type"
```

Comment:
we could not depend on l10n_es_edi_sii and deduce it from the extra fields we already it there?  I see only options 01-05 are really used.